protocol/localmessagenotification/README.md

```
@@ -0,0 +1,131 @@
+```
+
+## Usage Example
+
+```go
+// Non-blocking: get available messages immediately
+messages, hasMore := client.RequestMessages(blocking: false)
+
+// Blocking: wait for new messages
+messages, hasMore := client.RequestMessages(blocking: true)
+
+// Process received messages
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)